src/containerapp/azext_containerapp/_params.py

```diff
@@ -344,3 +344,9 @@ def load_arguments(self, _):
         c.argument('environment_name', options_list=['--environment'], help="The environment name.")
         c.argument('resource_group_name', arg_type=resource_group_name_type, id_part=None)
         c.argument('configuration', nargs="*", help="Java component configuration. Configuration must be in format \"<propertyName>=<value> <propertyName>=<value> ...\".")
+
+    with self.argument_context('containerapp env dotnet-component') as c:
+        c.argument('dotnet_component_name', options_list=['--name', '-n'], help="The DotNet component name.")
+        c.argument('environment_name', options_list=['--environment'], help="The environment name.")
+        c.argument('resource_group_name', arg_type=resource_group_name_type, id_part=None)
+        c.argument('dotnet_component_type', options_list = ['--type'], help="The type of DotNet component. Supported type is 'AspireDashboard'.")
```

Comment:
For supported type:
`, arg_type=get_enum_type(['AspireDashboard'])`

Reply:
Updated.

Reply:
Expectation is the following image. This is already being handled by CP
![image](https://github.com/Azure/azure-cli-extensions/assets/108305436/2c928314-6c5b-4695-b711-b9ec9df31211)

Reply:
![image](https://github.com/Azure/azure-cli-extensions/assets/46143499/95d1ff55-9d1c-4f9c-aaca-e21811c58d2a)
This is wired and not expected. Can you show me the code of the params you changed?

Reply:
@snehapar9 
 
It is a common recommend setting in CLI, and I tested:
![image](https://github.com/Azure/azure-cli-extensions/assets/46143499/a31dca79-1cff-4f4a-9636-a76f90536fd1)
It works fine:
`        c.argument('dotnet_component_type', options_list=['--type'], help="The type of DotNet component. Supported type is 'AspireDashboard'.", arg_type=get_enum_type(['AspireDashboard']))`

Reply:
![image](https://github.com/Azure/azure-cli-extensions/assets/46143499/18fb90d6-d23a-4ed7-9495-f1dcb0f9dc93)

Reply:
Okay, got it. I've updated it.